app/src/main/java/chat/rocket/android/renderer/MessageRenderer.java

```diff
@@ -123,18 +121,15 @@ public MessageRenderer urlsInto(RocketChatMessageUrlsLayout urlsLayout) {
       urlsLayout.setVisibility(View.VISIBLE);
       urlsLayout.setUrls(webContents, autoloadImages);
     }
-
     return this;
   }
 
   /**
    * show urls in RocketChatMessageUrlsLayout.
    */
-  public MessageRenderer attachmentsInto(RocketChatMessageAttachmentsLayout attachmentsLayout,
-                                         AbsoluteUrl absoluteUrl) {
-    if (!shouldHandle(attachmentsLayout)) {
+  public MessageRenderer attachmentsInto(RocketChatMessageAttachmentsLayout attachmentsLayout, AbsoluteUrl absoluteUrl) {
+    if (!shouldHandle(attachmentsLayout))
```

Comment:
Use braces for all `if` statements or use a single line
`if (!shouldHandle(attachmentsLayout)) return this;`